client/app/scripts/liveblog-edit/directives.js

```diff
@@ -159,7 +170,17 @@ define([
                         post: '=',
                         onEditClick: '=',
                         allowQuickEdit: '=',
-                        allowUnpublish: '='
+                        //the post that is in the process of being reordered
+                        reorderPost: '=',
+                        //the order property of the post that was reordered and should stay highlighted a bit more
+                        keepHighlighted: '=',
+                        allowUnpublish: '=',
+                        //call when the user clicks on the reorder icon 
+                        startReorder: '&',
+                        //call when the user has chosen a new place for the post
+                        reorder: '&',
+                        //the index of the post in the list
+                        index: '='
                     },
                     replace: true,
```

Comment:
The `replace` option has been deprecated, try to figure out how to get rid of it, if possible.